modules/ROOT/pages/authentication-authorization/database-administration.adoc

```diff
@@ -628,6 +677,172 @@ a|Rows: 6
 Note that `START` and `STOP` privileges are not included in the xref:authentication-authorization/database-administration.adoc#access-control-database-administration-all[`ALL DATABASE PRIVILEGES`].
 ====
 
+// TODO: mark this as available only in Cypher 25, and from 2025.07 (`ALTER DATABASE`, `SET DATABASE ACCESS`, `SET DATABASE DEFAULT LANGUAGE`) and 2025.08 (`ALTER COMPOSITE DATABASE`)
+[[access-control-database-administration-alter-db]]
+== The alter database related privileges
```

Comment:
how do I mark this as available only in Cypher 25 and from 2025.07 (for `ALTER DATABASE`, `SET DATABASE ACCESS`, and `SET DATABASE DEFAULT LANGUAGE`) and 2025.08 (for `ALTER COMPOSITE DATABASE`)?